.gitignore

```diff
@@ -1,9 +1 @@
-node_modules/
-/types/
-/firestore/
-/firebase/
-/next/
-/web/
-/helper.d.ts
-/*.js
-/*.ts
```

Comment:
The `*.js` file is the result of the build. So it's right to be included in the `.gitignore` list. So I think all the changes to this commit are inappropriate.
I used the method of building in the root folder for the purpose of using alias when distributing to npm, but I will modify it to distribute by folder using tools like [chakra-ui](https://github.com/chakra-ui/chakra-ui) later.